playbooks/robusta_playbooks/pod_enrichments.py

```diff
@@ -33,13 +34,24 @@ def pod_graph_enricher(pod_event: PodEvent, params: PodResourceGraphEnricherPara
     limit_lines = []
     if params.display_limits:
         resource_limits = pod_limits(pod)
+        request_limits = pod_requests(pod)
         if params.resource_type == "CPU" and resource_limits.cpu > 0:
             limit_line = XAxisLine(label="CPU Limit", value=resource_limits.cpu)
-            limit_lines = [limit_line]
+            limit_lines.append(limit_line)
+            if request_limits.cpu > 0:
```

Comment:
you can have a request without a limit so i would unindent this line
and change the if with
`if params.resource_type == "CPU" and resource_requests.cpu > 0:`

Reply:
done